powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/steps/v3/PrepareActivationStep.java

```diff
@@ -111,8 +113,8 @@ public JSONObject execute(StepLogger stepLogger, Map<String, Object> context) th
         Matcher m = p.matcher(model.getActivationCode());
         if (!m.find()) {
             if (stepLogger != null) {
-                stepLogger.writeError("Prepare activation step failed", "Activation code has invalid format");
-                stepLogger.writeDoneFailed();
+                stepLogger.writeError("activation-prepare-activation-code", "Prepare activation step failed", "Activation code has invalid format");
```

Comment:
I would use `create-activation` (or `activation-create`) prefix here. See my related comment in `v2/CreateActivationStep`.